gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -2345,8 +2358,12 @@ gst_tensor_filter_load_tensor_info (GstTensorFilterPrivate * priv)
     /** if set-property called and already has info, verify it! */
     if (prop->input_meta.num_tensors > 0) {
       if (!gst_tensors_info_is_equal (&in_info, &prop->input_meta)) {
-        g_critical ("The input tensor is not compatible.");
-        gst_tensor_filter_compare_tensors (&in_info, &prop->input_meta);
+        gchar *cmpstr =
+            gst_tensorsinfo_compare_to_string (&in_info, &prop->input_meta);
+        g_critical
```

Comment:
plz consider ml_loge(), not g_critical().

Reply:
fixed